docs/calicoctl/resources/bgppeer.md

```diff
@@ -37,4 +37,4 @@ spec:
 #### Spec
 | name     | description                 | requirements               | schema  |
 |----------|-----------------------------|----------------------------|---------|
-| asNumber | The AS Number of this peer. | Must be a valid AS Number. | integer |
+| asNumber | The AS Number of this peer. | Must be a valid AS Number.  The YAML or JSON format may be a string and may be specified in dotted notation. | integer |
```

Comment:
It works viewed in git (Which is currently all i'm worried about)

I ned to migrate this over to our docs - and they'll be a ton to fix up I figure - so i'm not too worried now, but I'd like the text at least to be accurate.